content/case_histories/rockglacier/index.rst

```diff
@@ -0,0 +1,36 @@
+.. _rockglacier_index:
+
+3D Helicopter GPR surveying a rock glacier
+==========================================
+
+.. raw:: html
+    :file: ../../../underconstruction.html
+
+- **Authors**: XXX
```

Comment:
before merging to master, could you please add the authors (or at least the one corresponding author)?